deploy/Deploy.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import { ALMProxy }          from "src/ALMProxy.sol";
+import { ForeignController } from "src/ForeignController.sol";
+import { MainnetController } from "src/MainnetController.sol";
+import { RateLimits }        from "src/RateLimits.sol";
+
+library ALMProxyDeploy {
```

Comment:
Can drop this and `RateLimitsDeploy`. I think the deploys should just be either foreign controller or mainnet controller and optionally do the alm proxy + rate limits.

Functions should be:

`ForeignControllerDeploy.deployController()` - will just do the controller for when we upgrade
`ForeignControllerDeploy.deployFull()` - will do ALM Proxy and Rates

and same for `MainnetControllerDeploy`